smart_lists/templates/smart_lists/smart_list.html

```diff
@@ -53,7 +53,7 @@
                         {% if forloop.first %}
                             <a href="{{ field.object.get_absolute_url }}" class="{{ table_link_class }}">{{ field.get_value }}</a>
                         {% else %}
-                            {{ field.get_value }}
+                            {{ field.get_value|safe }}
```

Comment:
Since this could be user-submitted content, this change opens up an XSS vulnerability. We should only call `safe` if the column was defined as a callable. In that case, the developer should implement proper escaping in their column (or most likely just use a django template).

Reply:
You are totally right.